ts/features/bonus/cgn/store/sagas/index.ts

```diff
@@ -0,0 +1,10 @@
+import { SagaIterator } from "redux-saga";
```

Comment:
In BPD feature we prefer this folders structure:

- feature
    - saga
    - store

instead of

- feature
    - store
        - saga

I find the first as a better solution since a saga is about _networking_, _orchestration_ or _navigation_. It is no strictly related to the store

What do you think?

Reply:
Fixed in b3f71c1